gubernator/templates/build.html

```diff
@@ -49,7 +49,9 @@
 		% for name, time, text, filename in failures
 			<hr>
 			<h3><a class="anchor" id="{{name|slugify}}" href="#{{name|slugify}}">{{name}}<span class="time"> {{time|duration}}</span></h3></a>
-			<pre class="cmd" onclick="select(this)">{{name | testcmd}}</pre>
+			% if 'junit_runner' not in filename
```

Comment:
I think the best way would be if it checked for "e2e.go" classname, but this is probably fine.